File: 3790-fruits-into-baskets-ii/3790-fruits-into-baskets-ii.cpp
class Solution {
public:
    int numOfUnplacedFruits(vector<int>& fruits, vector<int>& baskets) {
        int n = fruits.size();
        int cnt = n;
        for(auto& f : fruits) {
            for(int i = 0;i < n;i++){
                if(baskets[i] >= f){
                    baskets[i] = -1;
                    cnt--;
                    break;
                }
            }
        }
        return cnt;
    }
};

File: 3790-fruits-into-baskets-ii/3790-fruits-into-baskets-ii.java
class Solution {
    public int numOfUnplacedFruits(int[] fruits, int[] baskets) {
        int n = fruits.length;
        int unplaced = n;
        for(int i = 0;i < n;i++) {
            for(int j = 0;j < n;j++) {
                if(baskets[j] >= fruits[i]){
                    baskets[j] = -1;
                    unplaced--;
                    break;
                }
            }
        }
        return unplaced;
    }
}